.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @matrix-org/element-web
+* @matrix-org/element-call-reviewers
```

Comment:
this change needs reverting, unless you're planning on taking ownership for the whole js-sdk 😇

Reply:
I think we should perhaps take ownership of `/webrtc`?

Reply:
yeah, that would probably make sense.